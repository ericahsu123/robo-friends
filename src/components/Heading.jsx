import React from 'react';

const Heading = () => {
  return (
    <div className="item">
      <h1>RoboFriends</h1>
    </div>
  );
};

export default Heading;
