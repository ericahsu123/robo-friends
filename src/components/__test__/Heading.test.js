import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

import Heading from '../Heading';

test('<Heading/> matches snapshot', () => {
  const tree = renderer.create(<Heading />).toJSON();
  expect(tree).toMatchSnapshot();
});
