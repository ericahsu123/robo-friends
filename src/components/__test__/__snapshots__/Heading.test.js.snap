// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading/> matches snapshot 1`] = `
<div
  className="item"
>
  <h1>
    RoboFriends
  </h1>
</div>
`;
